README.md

```diff
@@ -1,73 +1,206 @@
+- `/scripts` - Includes the code that validates the `starter-kits.json` file
+
+## Table of contents
+
+- [How to run the scaffolding script](#how-to-run-the-scaffolding-script)
+- [Approval process for a new starter kit](#approval-process-for-a-new-starter-kit)
+- [Feature branches for starter kits](#feature-branches-for-starter-kits)
+- [Starter kit naming convention](#starter-kit-naming-convention)
+- [How to add a new starter kit](#how-to-add-a-new-starter-kit)
+- [Starter kit essentials](#starter-kit-essentials)
+  - [`package.json` requirements](#packagejson-requirements)
+  - [Pinned Dependency Versions](#pinned-dependency-versions)
+  - [.nvmrc files](#nvmrc-files)
+  - [Code formatting and linting](#code-formatting-and-linting)
+  - [Testing](#testing)
+  - [README file structures](#readme-file-structures)
+- [Essentials for Frontend kits](#essentials-for-frontend-kits)
+  - [State management](#state-management)
+  - [Styling](#styling)
+  - [Example Components](#example-components)
+  - [Storybook](#storybook)
+- [Essentials for Backend kits](#essentials-for-backend-kits)
+  - [Data Stores](#data-stores)
+  - [CORS configuration](#cors-configuration)
+  - [API Implementation and documentation](#api-implementation-and-documentation)
+- [Adding a starter kit to the website](#adding-a-starter-kit-to-the-website)
+- [Adding a starter kit to the CLI](#adding-a-starter-kit-to-the-cli)
+- [How to report an issue with the site or a starter kit](#how-to-report-an-issue-with-the-site-or-a-starter-kit)
+- [How to get started contributing to starter.dev](#how-to-get-started-contributing-to-starterdev)
 
 ## How to run the scaffolding script
 
-In the command line, run `npx @this-dot/create-starter` and you will see the list of available starter kits. Select the kit you are interested in and name your new project. Once the installation is complete, `cd` into your new project and install the dependencies(`npm install`, `yarn`, etc.) Now you are ready to get started with your brand new project.
+1. Run `npx @this-dot/create-starter` in the command line
+2. Select a starter kit
+3. Name your project
+4. `cd` into your new project
+5. Install the dependencies (`npm install`, `yarn`, etc.)
+
+## Approval process for a new starter kit
+
+If you have an idea for a new starter kit, please open up a new [issue](https://github.com/thisdot/starter.dev/issues/new?assignees=&labels=New+Issue%2C+feature&template=feature_request.yml&title=%5BFeature%5D%3A+) and provide all of the necessary details(framework, state management, styling, etc.) for the kit. Once the issue has been created, one of the core team members will get back to you.
```

Comment:
Please don't include this. This is strictly our internal process and shouldn't be a part of this project's main flow.